scripts/sd-update

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -e
+
+clear_files="server/Procfile server/app.py"
+use_branch='master'
+force=''
+quick=false
+mydir=`pwd`
+
+script_name=`basename $0`
+
+function show_help {
+    echo "$script_name [-h] [-f] [-b 'branch']"
+    echo -e "\t-h - show help"
+    echo -e "\t-f - force push to origin"
+    echo -e "\t-q - quick; skip updating requirements"
+    echo -e "\t-b 'branch' - pull from branch"
+}
+
+TEMP=`getopt -o hfqb: --long force,quick,branch: -n '$script_name' -- "$@"`
+eval set -- "$TEMP"
+
+tmpdir=`mktemp -d`
+
+function cleanup {
+    cd "$mydir"
+    for f in $clear_files; do
+        cp -f -R "$tmpdir/$f" "$mydir/$f"
+    done
+    rm -fr "$tmpdir"
+}
+
+trap cleanup EXIT
+
+while true; do
+    case "$1" in
+    -h) show_help; exit 0 ;;
+    -f|--force) force='--force'; shift ;;
+    -q|--quick) quick=true; shift ;;
+    -b|--branch) use_branch=$2; shift ;;
+    *) shift; break ;;
+    esac
+done
+
+for f in $clear_files; do
```

Comment:
I modify some files from the backend while working but I don't push these changes as they are not welcomed it seems. E.g. in app.py I always set debug = False (line 122). I don't want these changes to be overwritten, that's why I save these files in a temp dir.